commands/upgrade/__init__.py

```diff
@@ -75,8 +75,12 @@ def upgrade(args):
     with beautify_actor_exception():
         logger.info("Using answerfile at %s", answerfile_path)
         workflow.load_answers(answerfile_path, userchoices_path)
+
+        # Set the locale, so that the actors parsing command outputs that might be localized will not fail
+        os.environ['LANG'] = 'en_US.UTF-8'
         workflow.run(context=context, skip_phases_until=skip_phases_until, skip_dialogs=True,
                      only_with_tags=only_with_tags)
+        os.environ.pop('LANG')
```

Comment:
I guess you do not want to keep the pops :)

Reply:
You are right.

Lesson learned / retrospective: I guess this is a habit I've picked up in school. - _Programmer should clean the mess he created._ What was missing was focusing more at the context of whether we wish to restore the original state after we've finished with what we needed (that is to run the workflow with a specific locale). Thanks :+1:

Reply:
you are welcome. In this case we do not want to clean it as we first require people to use this locale anyway; second - the process above leapp is untouched by any changes we do inside the leapp, so after the execution it doesn't matter we we put inside.

Reply:
Fixed in the latest version of this PR. 
(Also added a proper commit message).